src/main/kotlin/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain
+
+import racingcar.domain.Location.Companion.BASE_LOCATION
+
+class Car(
+    private var _location: Location = BASE_LOCATION
+) {
+    val location: Location
+        get() = _location
```

Comment:
```suggestion
class Car(location: Location = BASE_LOCATION) {
    private var _location = location
    val location get() = _location
```
생성자에 언더바를 붙이는 것은 약간(?) 보기가 좋지 않아요. (비록 외부에서 접근은 하지 못하지만, 객체 생성 시 노출되기 떄문)
이런 경우에는 생성자를 프로퍼티로 만들지 않고, 클래스 내부에서 프로퍼티로 만들면 돼요.

Reply:
오 너무 좋은 방법이네요 ! 👍